common/djangoapps/student/forms.py

```diff
@@ -3,6 +3,9 @@
 """
 from importlib import import_module
 import re
+import socket
```

Comment:
@schenedx There was previous work done by @waheedahmed to completely remove these views, but I think there was push back from the community. Can you check with @rlucioni and @waheedahmed to get their take on the situation?